main.go

```diff
@@ -18,6 +18,7 @@
 package main
 
 //go:generate make -C .. -f cockroach/build/protobuf.mk
+//go:generate make storedeps
```

Comment:
Well we want to make sure `deps` (the file) doesn't drift, and we already enforce that everything is `go generate`d appropriately on every build, so it seemed like a convenient hook point. Where would you do it?